Add metadata flags for drift (#103)
aws-cloudformation/aws-cloudformation-resource-schema: README.md

| true           | true           | ordered set    |
| false          | true           | set      |

> #### optionPrefix/Suffix

Reviewer: just in case anyone is searching for documentation
```suggestion
#### optionalPrefix/optionalSuffix
```

Author: sure

> Some properties can be updated by the downstream service teams and these properties could have a prefix/postfix or both once returned from the Read Handler. These metadata properties are strings

Reviewer 2: ```suggestion
Some properties can be updated by the downstream service teams and these properties could have a prefix/suffix or both once returned from the Read Handler. These metadata properties are strings
```

> which can be provided for a property and checked on drift to ensure the property is not drifted.

Reviewer 2: ```suggestion
which can be provided for a property and checked on drift to ensure the property has not drifted.
```

`optionalPrefix: "anyString"`

> `optionalSuffix: '"anyString"`

Reviewer 3: Extraneous single quote

#### nullEqualsEmpty

> Some service teams consider null and empty list to be the same. This metadata property allows such properties to avoid drift when they equating null = []. This metadata takes in a bollean true|false.

Reviewer 4: ```suggestion
Some service teams consider null and empty list to be the same. This metadata property allows such properties to avoid drift when they equating null = []. This metadata takes in a boolean true|false.
```

By default drift considers null and empty list to be different.
`nullEqualsEmpty: true`

> #### caseInsesitive

Reviewer: typo
```suggestion
#### caseInsensitive
```

> Some service teams support properties which are case insensitive. This metadata property allows such properties to avoid drift when ignoring case. This metadata takes in a bollean true|false.

Reviewer 4: ```suggestion
Some service teams support properties that are case insensitive. This metadata property allows such properties to avoid drift when ignoring case. This metadata takes in a boolean true|false.
```

By default drift is case sensitive.

> `caseInsesitive: true`

Reviewer: typo
```suggestion
`caseInsensitive: true`
```
